specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -2333,23 +2333,21 @@
           "description": "Properties of the amlFilesystem.",
           "properties": {
             "storageCapacityTiB": {
-              "description": "The size of this filesystem, in TiB.  This may be rounded up.",
+              "description": "The size of this filesystem, in TiB.  This may be rounded up. This field can't be supplied in modify operations.",
               "type": "number",
-              "format": "float",
-              "readOnly": true
+              "format": "float"
             },
             "health": {
               "description": "Health of the amlFilesystem.",
               "readOnly": true,
-              "$ref": "#/definitions/amlFilesystemHealth"
+              "$ref": "#/definitions/AmlFilesystemHealth"
             },
             "zones": {
               "type": "array",
               "items": {
                 "type": "string"
               },
-              "description": "Availability zones for nodes.",
-              "readOnly": true
+              "description": "Availability zones for nodes. This field can't be supplied in modify operations."
```

Comment:
Can we say "resources" instead of "nodes"?

Reply:
Updated.